gallery/.eslintrc

```diff
@@ -10,5 +10,14 @@
   "rules": {
     "@typescript-eslint/no-var-requires": "off",
     "react/prop-types": "off"
-  }
+  },
+  "overrides": [
+    {
+      "files": ["*.tsx", "*.js", "*.ts"],
```

Comment:
This pattern doesn't seem right, shouldn't this be constrained to `js` files in the `visualtests` directory?

Reply:
> This pattern doesn't seem right, shouldn't this be constrained to `js` files in the `visualtests` directory?

Yeah, that makes sense: 2cf88baf5f34897fd2122a48e2d00899b5d835bd